en/components/grids_templates/column-hiding.md

```diff
@@ -202,15 +202,14 @@ For this purpose all we have to do is set both the [`IgxGridToolbarActionsDirect
 
 
 The @@igComponent provides us with some useful properties when it comes to using the toolbar's column hiding UI.
-By using the `igx-grid-toolbar-hiding` [`title`]({environment:angularApiUrl}/classes/igxgridtoolbarhidingcomponent.html#title) and the [`hiddenColumnsText`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#hiddencolumnstext) properties, we will set the title and the text that is displayed inside the dropdown button in the toolbar. More specifically, the [`hiddenColumnsText`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#hiddencolumnstext) is displayed right next to the columns count number, which is included by default in the button.
-The [`hiddenColumnsText`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#hiddencolumnstext) property is quite handy since it can be used for localization purposes as well.
+By using the `igx-grid-toolbar-hiding` [`title`]({environment:angularApiUrl}/classes/igxgridtoolbarhidingcomponent.html#title) property, we will set the title that is displayed inside the dropdown button in the toolbar.
 
 @@if (igxName === 'IgxGrid' || igxName === 'IgxTreeGrid') {
 ```html
 <!--columnHiding.component.html-->
 
 <div class="grid__wrapper">
-    <@@igSelector hiddenColumnsText="Hidden">
+    <@@igSelector>
         <igx-grid-toolbar>
             <igx-grid-toolbar-title>Employees</igx-grid-toolbar-title>
             <igx-grid-toolbar-actions>
```

Comment:
The same `hidingActionRef` should be added to the toolbar hiding component as well as it is missing when grid and tree grid, its visible only when having hierarchical-grid.
Grid and tree grid:
![image](https://user-images.githubusercontent.com/11193764/177990390-62da7ccc-dfc6-4a71-8024-1f655b7fc5a1.png)

HGrid:
![image](https://user-images.githubusercontent.com/11193764/177990481-3e4dc417-1c37-4f7c-b0bc-d4b89c0a7b37.png)